packages/dds/tree/src/feature-libraries/flex-map-tree/mapTreeNode.ts

```diff
@@ -104,8 +105,9 @@ export class EagerMapTreeNode<TSchema extends FlexTreeNodeSchema> implements Map
 		public readonly schema: TSchema,
 		/** The underlying {@link MapTree} that this `MapTreeNode` reads its data from */
 		public readonly mapTree: ExclusiveMapTree,
-		private location: LocationInField | undefined,
+		location: LocationInField | undefined,
```

Comment:
Should we just make the default value of this `unparentedLocation`?
```suggestion
		location: LocationInField = unparentedLocation,
```

Reply:
Yes, good call